src/renderer/mod.rs

```diff
@@ -81,11 +83,63 @@ impl<'a> Renderer<'a> {
             return Ok(to_value(to_string_pretty(context).unwrap()).unwrap());
         }
 
+        #[inline]
+        fn evaluate_sub_variable(context: &Value, key: &str, tpl_name: &str) -> String {
+            let sub_vars_to_calc = pull_out_square_bracket(key);
+            let mut new_key = key.to_string();
+
+            for sub_var in sub_vars_to_calc.iter() {
+                // Translate from variable name to variable value
+                let post_var = find_variable(context, sub_var.as_ref(), tpl_name).unwrap();
```

Comment:
why is this `unwrap`? I believe this can fail is the value is not present no?

Reply:
fixed in a refactor